bindings/wasm/package.json

```diff
@@ -0,0 +1,68 @@
+{
+    "name": "@iota/wallet-wasm",
+    "version": "1.0.0-alpha.1",
+    "description": "WebAssembly bindings for the IOTA wallet library",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/iotaledger/wallet.rs"
+    },
+    "author": "IOTA Foundation <contact@iota.org>",
+    "license": "Apache-2.0",
+    "publishConfig": {
+        "access": "public"
+    },
+    "files": [
+        "web/*",
+        "node/*",
+        "!web/test/",
+        "!node/test/"
+    ],
+    "scripts": {
+        "build": "npm run build:nodejs && npm run build:web",
+        "build:src": "cargo build --lib --release --target wasm32-unknown-unknown",
+        "build:nodejs": "node ./build_scripts/copyNeonDefs.js && npm run build:src && npm run bundle:nodejs && wasm-opt -O node/wasm/wallet_wasm_bg.wasm -o node/wasm/wallet_wasm_bg.wasm",
+        "build:web": "node ./build_scripts/copyNeonDefs.js && npm run build:src && npm run bundle:web && wasm-opt -O web/wasm/wallet_wasm_bg.wasm -o web/wasm/wallet_wasm_bg.wasm",
+        "bundle:nodejs": "wasm-bindgen target/wasm32-unknown-unknown/release/wallet_wasm.wasm --typescript --weak-refs --target nodejs --out-dir node/wasm && node ./build_scripts/node && tsc --project tsconfig.node.json --outDir node",
+        "bundle:web": "wasm-bindgen target/wasm32-unknown-unknown/release/wallet_wasm.wasm --typescript --weak-refs --target web --out-dir web/wasm && node ./build_scripts/web && tsc --project tsconfig.web.json --outDir web",
+        "copy-neon-defs": "node ./build_scripts/copyNeonDefs.js",
```

Comment:
Can you rename everything like we did in the client for consistency please ? neon -> nodejs